ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -1421,6 +1421,7 @@ public StartLeaderElectionReplyProto startLeaderElection(StartLeaderElectionRequ
         return ServerProtoUtils.toStartLeaderElectionReplyProto(leaderId, getMemberId(), false);
       }
 
+      leaderElectionMetrics.onLeaderElectionTimeout();
```

Comment:
onLeaderElectionTimeout() seems incorrect here.  Wrong method?

Reply:
@szetszwo When I used transferLeader, I found that this metric was not updated.
The meaning of this metric is election timeout, that is, when Follower -> Candidate, this metric value +1, so I think this modification should fix the problem, please take a look again, thx!

Reply:
Let's use a different method since this case is not "timeout".  How about adding a new TRANSFER_LEADERSHIP_COUNT_METRIC to LeaderElectionMetrics?

Reply:
@szetszwo Good idea, will do!